src/ert/__main__.py

```diff
@@ -261,6 +268,18 @@ def get_ert_parser(parser: Optional[ArgumentParser] = None) -> ArgumentParser:
     )
     FeatureToggling.add_feature_toggling_args(gui_parser)
 
+    # lint_parser
+    lint_parser = subparsers.add_parser(
+        "lint",
+        description="Lints an existing .ert config file.",
```

Comment:
Might want a more detailed description. It is not exactly the .ert config file as it looks at the entire setup. Also, the word lint could be expanded to "Checks for errors".

Reply:
Fixed, changed to: `Find and print errors in existing .ert configuration, including errors related to files used in the ert config.`